certbot/crypto_util.py

```diff
@@ -241,6 +243,71 @@ def valid_privkey(privkey):
     except (TypeError, OpenSSL.crypto.Error):
         return False
 
+def verify_renewable_cert_sig(renewable_cert):
+    """ Verifies the signature of a `storage.RenewableCert` object.
+    :param typ: `storage.RenewableCert`
+
+    :raises OpenSSL.crypto.Error if signature verification fails
+    """
+    with open(renewable_cert.chain, 'rb') as chain:
+        with open(renewable_cert.cert, 'rb') as cert:
+            chain = pyopenssl_load_certificate(chain.read())[0]
```

Comment:
I think you should unwrap the code a bit here and add error handling, for possible IOErrors with file open, and indexerror when getting chain. These will enable us to be more verbose in logging / printing info to user now or in the future, making it faster for the user to find out what's wrong exactly.